bigquery/tests/unit/test__pandas_helpers.py

```diff
@@ -667,17 +665,66 @@ def test_list_columns_and_indexes_with_multiindex(module_under_test):
 @pytest.mark.skipif(pyarrow is None, reason="Requires `pyarrow`")
 def test_dataframe_to_arrow_with_multiindex(module_under_test):
     bq_schema = (
-        schema.SchemaField("int_index", "INTEGER"),
-        schema.SchemaField("str_col", "STRING"),
+        schema.SchemaField("str_index", "STRING"),
+        # int_index is intentionally omitted, to verify that it's okay to be
+        # missing indexes from the schema.
+        schema.SchemaField("dt_index", "DATETIME"),
         schema.SchemaField("int_col", "INTEGER"),
         schema.SchemaField("nullable_int_col", "INTEGER"),
-        schema.SchemaField("str_index", "STRING"),
+        schema.SchemaField("str_col", "STRING"),
     )
-    df_data = collections.OrderedDict
-    dataframe = pandas.DataFrame(
-
+    df_data = collections.OrderedDict(
+        [
+            ("int_col", [1, 2, 3, 4, 5, 6]),
+            ("nullable_int_col", [6.0, float("nan"), 7.0, float("nan"), 8.0, 9.0]),
+            ("str_col", ["apple", "banana", "cherry", "durian", "etrog", "fig"]),
+        ]
+    )
+    df_index = pandas.MultiIndex.from_tuples(
+        [
+            ("a", 0, datetime.datetime(1999, 12, 31, 23, 59, 59, 999999)),
+            ("a", 0, datetime.datetime(2000, 1, 1, 0, 0, 0)),
+            ("a", 1, datetime.datetime(1999, 12, 31, 23, 59, 59, 999999)),
+            ("b", 1, datetime.datetime(2000, 1, 1, 0, 0, 0)),
+            ("b", 0, datetime.datetime(1999, 12, 31, 23, 59, 59, 999999)),
+            ("b", 0, datetime.datetime(2000, 1, 1, 0, 0, 0)),
+        ],
+        names=["str_index", "int_index", "dt_index"],
     )
-    assert False
+    dataframe = pandas.DataFrame(df_data, index=df_index)
+
+    arrow_table = module_under_test.dataframe_to_arrow(dataframe, bq_schema)
+
+    assert arrow_table.schema.names == [
+        "str_index",
+        "dt_index",
```

Comment:
Why is `int_index` omitted, here? It does not seem to clash with any of the column names.
(I might have misunderstood something)